tests/foreman/cli/test_environment.py

```diff
@@ -30,184 +30,196 @@
-        self.assertIn(env_name, [res['name'] for res in results])
-
-        # Update org and loc
-        new_org = entities.Organization().create()
-        Environment.update(
-            {
-                'location-ids': self.loc2.id,
-                'organization-ids': new_org.id,
-                'name': environment['name'],
-            }
-        )
-        env_info = Environment.info({'name': environment['name']})
-        self.assertIn(self.loc2.name, env_info['locations'])
-        self.assertNotIn(self.loc.name, env_info['locations'])
-        self.assertIn(new_org.name, env_info['organizations'])
-        self.assertNotIn(self.org.name, env_info['organizations'])
-        # Update name
-        new_env_name = gen_string('alpha')
-        Environment.update({'id': environment['id'], 'new-name': new_env_name})
-        env_info = Environment.info({'id': environment['id']})
-        self.assertEqual(env_info['name'], new_env_name)
-
-        # Delete
-        Environment.delete({'id': environment['id']})
-        with self.assertRaises(CLIReturnCodeError):
-            Environment.info({'id': environment['id']})
-
-    @pytest.mark.tier1
-    def test_negative_delete_by_id(self):
-        """Create Environment then delete it by wrong ID
-
-        :id: fe77920c-62fd-4e0e-b960-a940a1370d10
-
-        :expectedresults: Environment is not deleted
-
-        :CaseImportance: Medium
-        """
-        for entity_id in invalid_id_list():
-            with self.subTest(entity_id):
-                with self.assertRaises(CLIReturnCodeError):
-                    Environment.delete({'id': entity_id})
-
-    @pytest.mark.tier1
-    def test_negative_update_name(self):
-        """Update the Environment with invalid values
-
-        :id: adc5ad73-0547-40f9-b4d4-649780cfb87a
-
-        :expectedresults: Environment is not updated
-
-        """
-        environment = make_environment()
-        for new_name in invalid_values_list():
-            with self.subTest(new_name):
-                with self.assertRaises(CLIReturnCodeError):
-                    Environment.update({'id': environment['id'], 'new-name': new_name})
-                result = Environment.info({'id': environment['id']})
-                self.assertEqual(environment['name'], result['name'])
-
-    @pytest.mark.tier1
-    def test_positive_sc_params(self):
-        """Check if environment sc-param subcommand works passing
-        an environment id
-
-        :id: 32de4f0e-7b52-411c-a111-9ed472c3fc34
-
-        :expectedresults: The command runs without raising an error
-
-        """
-        # Override one of the sc-params from puppet class
-        sc_params_list = SmartClassParameter.list(
-            {
-                'environment': self.env['name'],
-                'search': 'puppetclass="{}"'.format(self.puppet_class['name']),
-            }
+
+@pytest.fixture(scope='module')
+def module_org():
```

Comment:
As with the API test module, you should consolidate fixture use here to not re-define the fixture.

Reply:
Done.